integration-cli/docker_cli_rmi_test.go

```diff
@@ -77,6 +77,37 @@ func TestRmiTag(t *testing.T) {
 	logDone("rmi - tag,rmi- tagging the same images multiple times then removing tags")
 }
 
+func TestRmiImgIDForce(t *testing.T) {
+	d := NewDaemon(t)
+
+	if err := d.StartWithBusybox(); err != nil {
+		t.Fatal(err)
+	}
+	imagesBefore, _ := d.Cmd("images", "-a")
+	d.Cmd("tag", "busybox", "utest:tag1")
+	d.Cmd("tag", "busybox", "utest:tag2")
+	d.Cmd("tag", "busybox", "utest/docker:tag3")
+	d.Cmd("tag", "busybox", "utest:5000/docker:tag4")
+	{
+		imagesAfter, _ := d.Cmd("images", "-a")
+		if strings.Count(imagesAfter, "\n") != strings.Count(imagesBefore, "\n")+4 {
+			t.Fatalf("before: %q\n\nafter: %q\n", imagesBefore, imagesAfter)
```

Comment:
Can you make this failure a bit clearer to someone who may run the tests but not understand the code yet? Something like "Expected four additional images after tagging; docker images shows: %q".  Not sure the `imagesBefore` is necessary for debugging the test.